### PR TITLE
Cata branch

### PR DESCRIPTION
**Pull request by HoteaCatalin**

### CODE REVIEW

File: Mini_DBMS/Mini_DBMS/Helpers/XMLOperation.cs [hunk 1]
```diff
@@ -37,7 +34,8 @@ public void WriteToFile(List<Database> databases)
         {
             XmlSerializer serializer = new XmlSerializer(typeof(List<Database>));
 
-            string folderPath = Environment.GetFolderPath(Environment.SpecialFolder.Desktop) + "//DBMS_test.xml";
+            //string folderPath = Environment.GetFolderPath(Environment.SpecialFolder.Desktop) + "//DBMS_test.xml";
```
Comment (chioreanadrian) [inline Mini_DBMS/Mini_DBMS/Helpers/XMLOperation.cs]: Remove commented code. It's unnecesarry.

File: Mini_DBMS/Mini_DBMS/Helpers/XMLOperation.cs [hunk 2]
```diff
@@ -18,8 +16,7 @@ public List<Database> ReadFromFile()
 
             var serializer = new XmlSerializer(typeof(List<Database>));
 
-            var currentDirectory = Environment.CurrentDirectory;
-            //var stringulet = Environment.GetFolderPath(Environment.CurrentDirectory);
```
Comment (chioreanadrian) [inline Mini_DBMS/Mini_DBMS/Helpers/XMLOperation.cs]: Remove commented code. It's unnecessary.

File: Mini_DBMS/Mini_DBMS/Helpers/XMLOperation.cs [hunk 3]
```diff
@@ -18,8 +16,7 @@ public List<Database> ReadFromFile()
...
+            var currentDirectory = Environment.CurrentDirectory; // cu asta ce e aici?
```
Comment (chioreanadrian) [inline Mini_DBMS/Mini_DBMS/Helpers/XMLOperation.cs]: I tried something, but I failed. Can be deleted.

File: Mini_DBMS/Mini_DBMS/Views/Home/Index.cshtml [hunk 4]
```diff
@@ -15,18 +15,26 @@
...
-                    </td>
-                </tr>
+            <tr>
+                <td>@database.Name</td>
+                <td>@database.Tables.Count</td>
+                <td>
+                    <input type="button" value="Add table" onclick="location.href='@Url.Action("CreateTable", "Home", new { databaseName = database.Name })'" />
+                </td>
+                <td>
+                    <input type="button" value="Add field" onclick="location.href='@Url.Action("CreateField", "Home", new { databaseName = database.Name })'" />
```
Comment (chioreanadrian) [inline Mini_DBMS/Mini_DBMS/Views/Home/Index.cshtml]: Why this button is on databases view?

**Review by chioreanadrian — CHANGES_REQUESTED** [review]

File: Mini_DBMS/Mini_DBMS/Views/Home/Index.cshtml (hunk 4 above)
Comment (HoteaCatalin) [reply Mini_DBMS/Mini_DBMS/Views/Home/Index.cshtml]: so you can add on specific db directly a specific table/ field to a table from that db

Comment (chioreanadrian) [reply Mini_DBMS/Mini_DBMS/Views/Home/Index.cshtml]: But this will not work because you don't have the table name for the new field

**Review by chioreanadrian — APPROVED** [review]

**Pull request merged**